components/brave_wallet/common/eth_sign_typed_data_helper_unittest.cc

```diff
@@ -213,13 +208,13 @@ TEST(EthSignedTypedDataHelperUnitTest, InvalidEncodedData) {
         invalid_json, base::JSON_PARSE_CHROMIUM_EXTENSIONS |
                           base::JSON_ALLOW_TRAILING_COMMAS);
     ASSERT_TRUE(invalid_value);
-    
+
     std::unique_ptr<EthSignTypedDataHelper> invalid_types_helper =
         EthSignTypedDataHelper::Create(invalid_value->GetDict().Clone(),
                                        EthSignTypedDataHelper::Version::kV4);
-  auto encoded_domain_v4 =
-      invalid_types_helper->EncodeData("Domain", *(data_dict.FindDict("name")));
-  ASSERT_FALSE(encoded_domain_v4);
```

Comment:
This should be `EXPECT_FALSE`, we should only use assertion to stop test from executing when critical condition is not met. And this is not the case and there is nothing left after the this line, it will also make all invalid cases untested after one failed.

Reply:
Resolved in https://github.com/brave/brave-core/commit/91b899f7ea313dcca20685a0f7555ea716331bb5

Reply:
Also, since this is more of an obvious one that I probably just missed when trying to learn our code guidelines. Do we have a good onboarding document or something that I can read through to make it easier to align PRs I write with some of the basic things like this. So far, the main thing I've been relying on is the linter because I didn't spot anything in the handbook about this aspect.